[2014-05-13 00:49:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/acegroup/public_html/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-13 00:59:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/acegroup/public_html/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-13 01:03:19] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/acegroup/public_html/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-13 01:03:25] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /home/acegroup/public_html/app/routes.php(409): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('solutions')
#2 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /home/acegroup/public_html/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-05-13 01:05:25] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/acegroup/public_html/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-13 01:05:31] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/acegroup/public_html/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-13 01:05:32] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/acegroup/public_html/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-13 01:05:39] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/acegroup/public_html/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-13 01:05:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/acegroup/public_html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/acegroup/public_html/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
